src/main/java/seedu/address/model/property/client/AskingPrice.java

```diff
@@ -66,7 +66,9 @@ public boolean equals(Object other) {
 
     @Override
     public int compareTo(AskingPrice another) {
-        return this.askingPrice.compareTo(another.askingPrice);
+        int price1 = Integer.parseInt(this.askingPrice.substring(1).replace(",", ""));
+        int price2 = Integer.parseInt(another.askingPrice.substring(1).replace(",", ""));
```

Comment:
i think according to the validation regex price can have cents. Should we still consider that for the price?

Reply:
fixed